modules/admin_manual/pages/useful_pages.adoc

```diff
@@ -0,0 +1,18 @@
+= Useful Pages
+:toc: right
+
+:description: When new to ownCloud Server, the following list of useful pages help getting up and running quickly.
+
+== Introduction
+
+{description}
+
+== Page List
+
+Note that this section always points to the latest server version available. In case you need a different version, select your topic and manually switch to one of the available.
```

Comment:
```suggestion
Note that this section always points to the latest server version available. In case you need a different version, select your topic and manually switch to one of the available ones.
```